frontend/src/components/messages/MessagesTab.vue

```diff
@@ -1,37 +1,35 @@
 <template>
   <v-row>
     <v-col sm="12" md="6" class="pa-0" :class="borderClass">
-      <v-row>
-        <v-col class="mt-3 ml-3">
-          <div>
-            <AppButton variant="text" @click="toggleNewRequestDialog()">
-              <v-icon class="icon" left>mdi-email-plus-outline</v-icon>
-              <span class="btn-label">New message</span>
-            </AppButton>
-            <AppButton variant="text" @click="toggleMarkUnreadButtonInMessageTable()">
-              <v-icon class="icon" left>mdi-email-outline</v-icon>
-              <span class="btn-label">Mark unread</span>
-            </AppButton>
-            <AppButton variant="text" class="btn-style" @click="toggleMarkReadButton()">
-              <v-icon class="icon" left>mdi-email-open-outline</v-icon>
-              <span class="btn-label">Mark read</span>
-            </AppButton>
-          </div>
+      <v-row class="mt-0 ml-0">
+        <v-col cols="12">
+          <AppButton size="small" class="new-message-button" @click="toggleNewRequestDialog()">
+            <v-icon left>mdi-email-plus-outline</v-icon>
+            New message
+          </AppButton>
+          <AppButton size="small" style="display: inline-block" :primary="false" class="mx-1" @click="toggleMarkUnreadButtonInMessageTable()">
```

Comment:
should we reuse "new-message-button" css class instead of inline style? we can rename that css class to "message-button"

Reply:
Yeah, I think I missed this one.